docs/source/conf.py

```diff
@@ -0,0 +1,20 @@
+# Configuration file for the Sphinx documentation builder.
+
+# -- Project information
+project = "datajudge"
+copyright = "(C) 2020-2021 QuantCo Inc."
```

Comment:
Shouldn't this be 2022 already? Or maybe remove the second year. See [this](https://hynek.me/til/copyright-years/).